### ISSUE TITLE
hotcue colors have inconsistent brightness

### ISSUE THREAD

**Issue by m0dB** [mixxxdj/mixxx#12206]
### Bug Description

The hotcue colors are inconsistent in there brightness, as a result some are considered a dim color and others are not (according to isDimColorCustom). As a result, some have a white text in there default state and black in there hovered state, and others vice versa. Furthermore, this inconsistency has more consequences, e.g. with the line vs border line drawn in the waveform widget.

**Comment by daschuer** [comment]
What is the issue you like to have fixed with this bug report? 

The different brightness for our default cue colors is picked by design. They where a result from some tests and considerations regarding color blindness, to optimal distinguish them.

**Comment by ronso0** [comment]
> As a result, some have a white text in there default state and black in there hovered state

I agree this is distracting (never used the mouse in the waveforms).
Would be better to not invert colors on hover.

Alternative:
* only brighten the bg color if possible? Edge case, obviously: white and black
* don't draw the border / transparent border (reserve space, and draw it [cue color | brighter color] on hover?

**Comment by ronso0** [comment]
Hotcue buttons are not affect AFAICT. There, only the bg colors changes on hover if the skin sets `<Hover>true</Hover>` in the `HotCueButton` template. So IIUC the fix would be to simply:tm: adopt that for the waveform markers.

**Comment by m0dB** [comment]
Thanks for the info @ronso0

> What is the issue you like to have fixed with this bug report?

I think primarily that the letters on the hotcue marks are "randomly" white or black 

> Hotcue buttons are not affect AFAICT

I am using LateNight palemoon (which is the default, right?) and it does affect the buttons.

<img width="105" alt="Screenshot 2023-10-22 at 02 44 08" src="https://github.com/mixxxdj/mixxx/assets/79429057/b07db3cc-2462-4800-8c38-494645bf169b">

But looking at the skin, I am thinking that maybe this can be fixed by changing the dim bright threshold for the marks / the buttons?
